### ISSUE TITLE
Prestação de contas

### ISSUE THREAD

**Issue by wagner-sousa** [s3cm44s/osintt001#10]
Sobre a prestação de contas há algo incomum?

[Prestação de contas-20220927T122916Z-001.zip](https://github.com/s3cm44s/osintt001/files/9655818/Prestacao.de.contas-20220927T122916Z-001.zip)

**Comment by cagartner** [comment]
Essa prestação de contas foi ela que enviou? Pra mim não vale nada sem as fotos das notas fiscais relacionadas com os débitos do Nubank, quem garante que ela não comprou bebidas e comidas para ela e não para o projeto? Eu já fui subsindico e uma prestação mensal de condomínio é mais detalhada haha

**Comment by cagartner** [comment]
Outra coisa, a prestação de contas precisa ser feita desde o começo do projeto, não apenas do mês de setembro, e ainda precisaria ter a informação completa do extrato, não apenas um print do resumo.

E ainda tem os gastos do cartão: 
![image](https://user-images.githubusercontent.com/4007696/192529969-d0377d7d-9034-40d1-acd6-70c3d78912bb.png)

O que são esses gastos? Porque todas as notas são em débitos e não no crédito, vai precisar esclarecer tudo isso porque tudo leva a crer que ela usa a conta pessoal dela para o projeto, vai ser complicado conseguir explicar e comprovar que ela não usou dinheiro do projeto para o bem próprio.

**Comment by wagner-sousa** [comment]
> Essa prestação de contas foi ela que enviou? Pra mim não vale nada sem as fotos das notas fiscais relacionadas com os débitos do Nubank, quem garante que ela não comprou bebidas e comidas para ela e não para o projeto? Eu já fui subsindico e uma prestação mensal de condomínio é mais detalhada haha

Foi postada por ela mesmo hoje, segundo ela também será postado as Notas Fiscais.

**Comment by joaovictorsouza** [comment]
A prestação não faz sentido a partir do momento que ela alega ter "poupanças" que estão definidas como reservado para limite de crédito, mas o print mostra que os valores não foram para o crédito no cartão. 
Ela provavelmente foi movimentando o dinheiro de lugar e tirando prints para criar a narrativa de que o valor está com ela.

**Comment by wagner-sousa** [comment]
> A prestação não faz sentido a partir do momento que ela alega ter "poupanças" que estão definidas como reservado para limite de crédito, mas o print mostra que os valores não foram para o crédito no cartão. Ela provavelmente foi movimentando o dinheiro de lugar e tirando prints para criar a narrativa de que o valor está com ela.

Poderia explicar melhor?

**Comment by joaovictorsouza** [comment]
Explico sim.
O Nubank possui uma ferramenta onde você guarda um dinheiro (reserva um valor) e esse valor se torna limite de crédito no cartão de crédito. Aí você usa esse limite para comprar, parcelamento e etc. 
E você pode resgatar o dinheiro se não tiver comprometido o limite que ele te deu. 

Se ela está usando essa ferramenta guardando aqueles dois valores que ela cita ter guardado (inclusive nem sei se dá pra guardar dois montantes separados nessa ferramenta), esse valor teria que refletir no crédito. 

Se eu reservo 5 mil reais o limite de crédito no cartão aumenta em 5 mil reais. 
Como ela tem tanto armazenado nessa função e o limite praticamente zerado?!


Das duas uma: ela forjou esses prints ou então ela tem isso armazenado mas gastou na forma de crédito todo esse valor, pra não estar constando limite livre. E o valor da fatura seria fatura atual desconsiderando parcelamentos.

**Comment by cagartner** [comment]
Outra opção é que ela diminuiu o limite, no Nubank, vc pode ter um limite de 5mil mas apenas liberar 1 mil de limite para evitar golpes ou compra indesejada. Mas mesmo assim é muito estranho.

**Comment by joaovictorsouza** [comment]
> Outra opção é que ela diminuiu o limite, no Nubank, vc pode ter um limite de 5mil mas apenas liberar 1 mil de limite para evitar golpes ou compra indesejada. Mas mesmo assim é muito estranho.
> 

Sim, tem essa possibilidade. Mas é estanha porque se a intenção era apenas guardar dinheiro há outra ferramenta no nubank que não estende o limite e tem rentabilidade maior. Acho que na pressa ela nem viu onde tava colocando. 
E ela mostra as duas supostas poupanças em prints recortados e separados, provavelmente ela só colocou o dinheiro, tirou o print. Diminuiu o valor e tirou outro. 
Até porque acho que nessa opção de reserva pra limite nem tem como guardar duas quantias separadas. Pra isso o Nubank disponibiliza a ferramenta de caixinhas.

**Comment by Mike-Null** [comment]
Os "comprovantes de compras no débito" mostram MUITOS sinais que sugerem terem sido forjados. Vou postar só alguns exemplos de como regiões diferentes das imagens se destacam visivelmente em uma análise de ruídos.

1. comprovante com pouca ou nenhuma alteração:

![12](https://user-images.githubusercontent.com/114615977/192871438-f0195751-dfd4-4d19-9b9f-a206d286dc6c.jpg)

2. comprovante com FORTE indício de apenas o nome do supermercado ter sido copiado e colado ali:
![1](https://user-images.githubusercontent.com/114615977/192871926-186742f3-3c95-4936-973e-f4963e463f15.jpg)

3. um par de comprovantes que parecem ter sido editados _toscamente_, só mudando o valor da compra e a data entre eles:
![8](https://user-images.githubusercontent.com/114615977/192872079-c95dd571-ae99-4063-9bf3-b401ca04a553.jpg)
![9](https://user-images.githubusercontent.com/114615977/192872199-e493693b-8adb-4b8a-9c08-d2790247dd1b.jpg)

Tem outros indícios bem claros além desses, mas talvez seja melhor guardar umas cartas na manga pro caso dela continuar tentando jurar de pés juntos que as imagens são verdadeiras. Tudo que eu citei é fácil de analisar com ferramentas gratuitas ou mesmo softwares simples de edição de imagem.

**Comment by wagner-sousa** [comment]
@Mike-Null 
> Tem outros indícios bem claros além desses, mas talvez seja melhor guardar umas cartas na manga pro caso dela continuar tentando jurar de pés juntos que as imagens são verdadeiras. Tudo que eu citei é fácil de analisar com ferramentas gratuitas ou mesmo softwares simples de edição de imagem.

Acho até melhor postar aqui mesmo, para auxiliar o MP e eles usarem como prova, não?
Ela pode até tentar negar, mas, ela tem muito mais coisas para se preocupar.

**Comment by Mike-Null** [comment]
OK, essa acho que é bem simples de visualizar: a posição do nome do supermercado varia em alguns pixels para cima ou para baixo dependendo da imagem. Perceba como ele parece "flutuar" quando se alterna rápido entre elas.

![posição do nome do comprador](https://user-images.githubusercontent.com/114615977/192921970-4323e24c-769a-47a9-a154-01f255ff7713.gif)

**Comment by wagner-sousa** [comment]
Muito bom, parabéns. Dois ponto:
* É impressão minha ou a tonalidade do roxo altera?
* Ainda há um argumento de defesa que é o horário das fotos no fundo

**Comment by Mike-Null** [comment]
> É impressão minha ou a tonalidade do roxo altera?

Sim, muda o tom mesmo entre imagens que supostamente foram capturadas no mesmo minuto (10:14). 

> Ainda há um argumento de defesa que é o horário das fotos no fundo

Como isso funcionaria como argumento de defesa?

**Comment by wagner-sousa** [comment]
> Como isso funcionaria como argumento de defesa?
@Mike-Null 
Ela pode argumentar que são mais de um horário, mas já é de grande ajuda. Obrigado.

**Comment by joaovictorsouza** [comment]
Na hora que cair na justiça eles vão querer ver extrato do banco certinho, isso se não pedirem quebre de sigilo pra fazer investigação. Até porque o caso pegou repercussão, tem ministério público em cima agora...
Ela tá com um grande problema na mão caso sejam editados, porque se o que a alegou aqui não bater podem até encaixar isso como falsificação de documentos particular. 

Aí já junta que como ela supostamente se passou por uma outra pessoa (a tal da Taynara). É um conjunto de supostos crimes aí. 

Se isso não for loucura da nossa cabeça, a situação tá horrenda.
